src/Fight.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Fight.h"
+
+void PhysicsSystem::on_tick()
+{
+	for (auto&& [enemy_entity, enemy, pos] : QueryEnemies::access_storage().each())
+	{
+		for (auto&& [player_entity, player, player_pos] : QueryPlayers::access_storage().each())
+		{
+			if (intersects( geometry::Vec2{player_pos.xy.x,player_pos.xy.y},
+					geometry::Vec2{pos.xy.x + 100,pos.xy.y + 100}, 30))
```

Comment:
Interesting way to align :)
I prefer not to move the upper line that's already correctly placed. Instead, in the second line you can add spaces instead of the last tab:
```suggestion
			if (intersects(geometry::Vec2{player_pos.xy.x,player_pos.xy.y},
                                       geometry::Vec2{pos.xy.x + 100,pos.xy.y + 100}, 30))
```